Sources/URLEncodedForm/Codable/URLEncodedFormEncoder.swift

```diff
@@ -11,6 +11,23 @@
 /// See [Mozilla's](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/POST) docs for more information about
 /// url-encoded forms.
 public final class URLEncodedFormEncoder: DataEncoder {
+    /// The formatting of the output data.
+    public struct OutputFormatting : OptionSet {
+        /// The format's default value.
+        public let rawValue: UInt
+        
+        /// Creates an OutputFormatting value with the given raw value.
+        public init(rawValue: UInt) {
+            self.rawValue = rawValue
+        }
+        
+        /// Produce output with dictionary keys sorted in lexicographic order.
+        public static let sortedKeys = OutputFormatting(rawValue: 1 << 1)
+    }
+
+    /// The output format to produce. Defaults to `[]`.
+    public var outputFormatting: OutputFormatting = []
```

Comment:
Vapor style is to set all properties in the init method. We try to avoid default properties on members since they can be easy to miss when refactoring.